docs/usage.rst

```diff
@@ -308,21 +308,23 @@ the following launch configuration:
 
 .. code:: json
 
-   {
-       "version": "0.2.0",
-       "configurations": [
-           {
-               "name": "Attach to SAM CLI",
-               "type": "node",
-               "request": "attach",
-               "address": "localhost",
-               "port": 5858,
-               "localRoot": "${workspaceRoot}",
-               "remoteRoot": "/var/task",
-               "protocol": "legacy"
-           }
-       ]
-   }
+	{
+		"version": "0.2.0",
+		"configurations": [{
+	        "name": "Attach to SAM CLI",
+	        "type": "node",
+	        "request": "attach",
+	        "address": "localhost",
+	        "port": 5858,
+	        "localRoot": "${workspaceRoot}",
+	        "sourceMapPathOverrides": {
+	            "${workspaceRoot}/runtime": "${remoteRoot}"
+			    },
+	        "remoteRoot": "/var/task",
+	        "protocol": "inspector",

```

Comment:
The previous protocol was legacy. What does updating this to inspector indicate?